packages/blade/docs/tokens/Motion.stories.mdx

```diff
@@ -2,7 +2,7 @@ import { Meta, DocsContainer } from '@storybook/addon-docs';
 import styled from 'styled-components';
 import { useTheme, BladeProvider } from '../../src/components';
 import { paymentTheme, bankingTheme } from '../../src/tokens';
-import makeMotionTime from '../../src/utils/makeMotionTime';
+import { makeMotionTime } from '../../src/utils';
```

Comment:
can't we change paths here too?

Reply:
No it fails. Storybook doesn't resolve the paths in the MDX files.